src/js/11-i18n.js

```diff
@@ -3,9 +3,57 @@
 const translations = {
   'en-gb': {
     results_label: 'results',
+    components_title: 'Produkt filter',
+    manual: 'plentymarkets',
+
+    modules_title: 'Module filter',
+
+    '': 'General',
```

Comment:
I would avoid empty keys. Instead of this you could check for empty strings when making the translation and pass a default key instead.